src/Umbraco.Core/GuidUdi.cs

```diff
@@ -34,7 +34,7 @@ public GuidUdi(Uri uriValue)
         {
             Guid guid;
             if (Guid.TryParse(uriValue.AbsolutePath.TrimStart('/'), out guid) == false)
-                throw new FormatException("Url \"" + uriValue + "\" is not a guid entity id.");
+                throw new FormatException("URI \"" + uriValue + "\" is not a GUID entity id.");
```

Comment:
'id' is probably also an abbreviation of 'identifier', so this can be updated to:
```suggestion
                throw new FormatException("URI \"" + uriValue + "\" is not a GUID entity ID.");
```